src/main/java/com/google/api/codegen/transformer/csharp/CSharpCommonTransformer.java

```diff
@@ -102,6 +102,7 @@ public ParamWithSimpleDoc makeParam(
         .typeName(typeName)
         .setCallName("")
         .addCallName("")
+        .getCallName("")
         .isMap(false)
         .isArray(false)
         .isPrimitive(false)
```

Comment:
Not directly related to your PR, but I think here we should create a Type enum (MAP, ARRAY, PRIMITIVE) instead of having 3 flags.